index.html

```diff
@@ -241,11 +241,21 @@ <h3 class="form-section-content-h3">
             </li>
           </ul>
         </form>
+          <!-- <footer>
+          <ul>
+            <li><img src="./media/svg/github.svg" alt="Github icon"></li>
+            <li><img src="./media/svg/linkedin.svg" alt="Linkedin icon"></li>
+            <li><img src="./media/svg/angel.svg" alt="Angelist icon"></li>
+            <li><img src="./media/svg/twitter.svg" alt="Twitter icon"></li>
+            <li><img src="./media/svg/medium.svg" alt="Medium">
+            </li>
+          </ul>
+        </footer> -->
```

Comment:
- [ ]  Kyndly remove the commented code as requested on the previous review.This will make your code look more professional. 📚📚📈